system/aws-ecr-creds-helper/alerts/ecr-creds-helper.alerts

```diff
@@ -0,0 +1,17 @@
+groups:
+- name: ecr-creds-helper.alerts
+  rules:
+
+    ### ecr-creds-helper ###
+
+    - alert: EcrCredsHelperJobFailed
+      expr: sum(kube_job_status_failed{job_name=~"aws-ecr-creds-helper-.+"}) by (job_name) > 2
```

Comment:
summarizing by `job_name` is not correct as every job created by the cronjob has its own name. I think we want to summarise over multiple cronjob runs.
Maybe its also better to turn the query around and alert if there are no successful runs in the recent past.
E.g.:
```
sum(increase(kube_job_status_succeeded{job_name=~"aws-ecr-creds-helper-.+"}[8h])) == 0 
```